imagedephi/redact/redact.py

```diff
@@ -163,6 +166,50 @@ def redact_images(
             writer.writerow(row)
 
 
+def _custom_sort(item):
+    key, value = item
+    if key == "missing_tags":
+        return (0, key)
+    elif isinstance(value, dict) and value["action"] == "delete":
+        return (1, key)
+    else:
+        return (2, key)
+
+
+def _sort_data(data):
+    """
+    Sort images based on the presence of missing tags and then by image name.
+
+    Sort tags within each image based on the action and tag name.
+    """
+    global tags_used
+    sorted_data = {}
+    # List of tags that can't be edited and should be excluded from the redaction plan
+    excluded_tags = [
+        "StripOffsets",
+        "StripByteCounts",
+        "FreeOffsets",
+        "FreeByteCounts",
+        "TileOffsets",
+        "TileByteCounts",
+        "JPEGIFOffset",
+        "JPEGIFByteCount",
+    ]
```

Comment:
We should exclude showing `BigTIFF`, which isn't a tag but a format detail.

Reply:
done in 808131bfef0434c339f8feb03d57c83834784114